website/content/docs/api-clients/api.mdx

```diff
@@ -12,6 +12,7 @@ Boundary's API is a JSON-based HTTP API that adheres to a set of standards that
 Before reading this page, it is useful to understand Boundary's [domain model](/docs/concepts/domain-model) to be aware of the terminology used here.
 
 Boundary's API is also described via OpenAPI v2; the version corresponding to any tag of Boundary's source code can be found in Boundary's [GitHub repository](https://github.com/hashicorp/boundary/blob/main/internal/gen/controller.swagger.json).
+A rendered version of this generated API definition can be found on the [API page](/api-docs).
```

Comment:
If you want the link to visually stand out, you can make it into a callout note (use `->`)?  

![image](https://user-images.githubusercontent.com/7660718/190678975-4985e42a-0d6c-46f5-ab61-788f6037df20.png)

Just a thought.  Feel free to ignore this comment if you don't like the idea 😅 

```suggestion

-> **NOTE:** A rendered version of this generated API definition can be found on the [API page](/api-docs).
```